components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+set -u
+set -o pipefail
+
+usage(){
+	echo "Usage: $0 -n <deployment name> [-r <region>]"
+	exit 1
+}
+
+cwd=$(dirname $(realpath $0))
+
+### Input parameters
+DEPLOY_NAME="sagemaker-kfp-"$(date '+%Y-%m-%d-%H-%M-%S')"" # The name given to the entire deployment (tagging all resources)
+REGION=${REGION:-"$(aws configure get region)"} # Deployment region
+
+### Configuration parameters
+EKS_EXISTING_CLUSTER=${EKS_EXISTING_CLUSTER:-""} # Use an existing EKS cluster
+EKS_CLUSTER_VERSION=${EKS_CLUSTER_VERSION:-"1.15"} # EKS cluster K8s version
+EKS_NODE_COUNT=${EKS_NODE_COUNT:-"1"} # The initial node count of the EKS cluster
+EKS_PUBLIC_SUBNETS=${EKS_PUBLIC_SUBNETS:-""}
+EKS_PRIVATE_SUBNETS=${EKS_PRIVATE_SUBNETS:-""}
+
+### Testing parameters
+MINIO_LOCAL_PORT=${MINIO_LOCAL_PORT:-9000}
+KFP_NAMESPACE=${KFP_NAMESPACE:-"kubeflow"}
+KFP_SERVICE_ACCOUNT=${KFP_SERVICE_ACCOUNT:-"pipeline-runner"}
+
+PYTEST_MARKER=${PYTEST_MARKER:-""}
+S3_DATA_BUCKET=${S3_DATA_BUCKET:-"kfp-test-data"}
```

Comment:
it shouldn't be a problem, but might be better to not have this default here.

Reply:
Yeah fair enough. I might actually make it empty and if there is no value given throw an error before we even start the test process.